src/routes/(docs)/Toaster/ExampleToaster.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+	import { Toaster, showToast } from '$lib/toaster';
```

Comment:
add an export to the lib/index.ts file and then:
```suggestion
	import { Toaster, showToast } from '$lib';
```